pywt/data/_readers.py

```diff
@@ -90,6 +90,19 @@ def camera():
     camera : ndarray
        convenient image to use for testing and demonstration
 
+    Notes
+    -----
+    No copyright restrictions. CC0 by the photographer (Lav Varshney).
```

Comment:
The name of the photographer looks off, both here and in the scikit-image PR. It says "Lav" here, "Lev" in the PR description, and the GitHub profile linked from the scikit-image PR says "Kush R. Varshney" (https://github.com/krvarshney/krvarshney.github.io)

Reply:
I think Kush is the subject of the photograph, but the photographer is his brother, Lav (or Lev?). The scikit-image docstring says "Lav", but maybe that is a mistake. will check...

Reply:
@stefanv or @jni.  We are also replacing cameraman here. Can you confirm if the photographer's name is Lev or Lav?

Reply:
Lav. http://www.varshney.csl.illinois.edu

Reply:
Great, thanks @jni